vm/evaluation_test.go

```diff
@@ -786,3 +786,60 @@ func TestMethodCallWithNestedBlock(t *testing.T) {
 		testIntegerObject(t, evaluated, tt.expected)
 	}
 }
+
+func TestMethodCallWithoutParens(t *testing.T) {
+	tests := []struct {
+		input    string
+		expected int
+	}{
+		{
+			`
+			class Foo
+			  def set_x x0
+			    @x = x0
+			  end
+
+			  def foo
+			    set_x(10)
+			    a = 10
+			    @x + a
+			  end
+			end
+
+			f = Foo.new
+			f.foo
+			`,
+			20,
+		},
+		{
+			`
+			class Foo
+			  def set_x x1, x2, x3
+			    @x = x1
+			    @y = x2
+			  end
+
+			  def foo
+			    set_x(10,11,12)
```

Comment:
Actually you can use `attr_reader` here, then you can remove this method and just write:

```ruby
f.x + a + f.y